CHANGES/8481.bugfix.rst

```diff
@@ -0,0 +1,2 @@
+Fixed the incorrect rejection of ws:// and wss:// urls.
```

Comment:
Syntax fixes:
```suggestion
Fixed the incorrect rejection of ``ws://`` and ``wss://`` urls
```
But also, rewrite this to include context and the effect it has that the end-user audience would be able to understand. Currently, it's rather hard for somebody to interpret how this phrase would affect them.